processor/src/test/java/org/mapstruct/ap/test/bugs/_913/Issue913SetterMapperForCollectionsTest.java

```diff
@@ -173,6 +175,7 @@ public void shouldReturnDefaultForNvmsReturnDefaultForUpdate() {
         Dto dto = new Dto();
         Domain domain = new Domain();
         Set<Long> longIn = new HashSet<Long>();
+        longIn.add( 10L );
```

Comment:
It is just an improvement. You didn't miss anything, the test passed without changing anything. I just added this, to be sure that clear is called. The reason is that we check if the `longs` are empty and if they are empty from the start, you don't know if it is because empty was added or because of before :)